plugin/storage/es/dependencystore/storage.go

```diff
@@ -121,17 +122,12 @@ func buildTSQuery(endTs time.Time, lookback time.Duration) elastic.Query {
 	return elastic.NewRangeQuery("timestamp").Gte(endTs.Add(-lookback)).Lte(endTs)
 }
 
-func (s *DependencyStore) getIndices(ts time.Time, lookback time.Duration) []string {
+func getIndices(ts time.Time, lookback time.Duration) []string {
 	var indices []string
 	indices = append(indices, indexName(ts))
 	for lookback > 0 {
```

Comment:
no this just collects the indices that we're going to query over. it'll only retrieve the ones over the last hour because the query is a range query. the query is on 
https://github.com/uber/jaeger/blob/es-search-ignore-indices/plugin/storage/es/dependencystore/storage.go#L101
https://github.com/uber/jaeger/blob/es-search-ignore-indices/plugin/storage/es/dependencystore/storage.go#L121